src/tools/devtools/release-notes/release-notes-2.23.1-src.md

```diff
@@ -54,6 +54,6 @@ Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 * Fix a bug where JSON requests were shown in plain text, instead of the formatted JSON viewer. [#5463](https://github.com/flutter/devtools/pull/5463)
 * Fix a UI issue where the copy button on the response or request tab would let you copy while still loading the data. [#5476](https://github.com/flutter/devtools/pull/5476)
 
-## Changelog
-More details about changes and fixes are available in the DevTools
-[changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md).
\ No newline at end of file
+## Full commit history
+More details about changes and fixes are available from the
+[DevTools git log.](https://github.com/flutter/devtools/commits/master).
```

Comment:
**issue:** Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice). Use site variable.
```suggestion
To learn more about DevTools changes and fixes, check out the
[commit history]({{site.github}}/flutter/devtools/commits/master).
```